tests/test_cli/test_force_unlock.py

```diff
@@ -0,0 +1,57 @@
+"""Test suite for matcha cli force-unlock command."""
+from unittest.mock import patch
+from typer.testing import CliRunner
+
+from matcha_ml.cli.cli import app
+
+INTERNAL_FUNCTION_STUB = "matcha_ml.core.core"
+
+
+def test_cli_force_unlock_command_help_option(runner: CliRunner) -> None:
+    """Tests the --help option for the cli force-unlock command.
+
+    Args:
+        runner (CliRunner): typer CLI runner
+    """
+    # Invoke force-unlock command with help option
+    result = runner.invoke(app, ["force-unlock", "--help"])
+
+    # Exit code 0 means there was no error
+    assert result.exit_code == 0
+
+    # Assert string is present in cli output
+    assert "Force unlock remote matcha state on Azure." in result.stdout
```

Comment:
Failing test due to the updated docstring in force unlock command

Reply:
Updated thanks!